apps/ejabberd/src/gen_mod_deps.erl

```diff
@@ -0,0 +1,233 @@
+%%==============================================================================
```

Comment:
[100% coverage](https://coveralls.io/builds/9164574/source?filename=apps%2Fejabberd%2Fsrc%2Fgen_mod_deps.erl) for the new module! I'm impressed!